### ISSUE TITLE
simulate deferred acceptance algorithm N times 

### ISSUE THREAD

**Issue by j-stat** [j-stat/DApropensity.jl#7]
After #4 #8 #9 and #10 after complete, we need to produce a function to simulate the deferred acceptance algorithm N times with a single randomly generated lottery number (drawn from uniform distribution) assigned to each student for each single run of the N runs.
